hashes/src/hkdf/mod.rs

```diff
@@ -106,6 +106,31 @@ where
     }
 }
 
+impl<T: GeneralHash> fmt::Debug for Hkdf<T> {
+    fn fmt(&self, f: &mut ::core::fmt::Formatter) -> ::core::fmt::Result {
+        use crate::{sha256t, sha256t_tag};
+
+        sha256t_tag! {
+            pub struct Tag = hash_str("bitcoin_hashes1DEBUG");
+        }
+        let hash = sha256t::Hash::<Tag>::hash(self.prk.as_ref());
+        let mut buf = [0_u8; BYTES];
+        buf.copy_from_slice(&hash.as_byte_array()[..BYTES]);
```

Comment:
In d8c569350138c8084dfa65de074598325279a3fb:

You can construct `buf` directly with `let buf = core::array::from_fn(|i| hash.as_ref()[i])`.

And arguably you can inline `buf` into the constructor of `Fingerprint` ... and inline the constructor of `fingerprint` into the call to `format_args`.